game.c

```diff
@@ -108,13 +108,7 @@ void setup_level() {
         mobs[i].stacks = false;
     }
 
-    player = &mobs[MAX_MOBS-1];
```

Comment:
How is the player mob being defined? Doesn't this straight up break the game?

Reply:
If this was a question for me, then this is because I screwed up my repo. I didn't write that line.